pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/BrokerOperabilityMetrics.java

```diff
@@ -84,7 +84,7 @@ Map<String, String> getDimensionMap(String metricsName) {
     }
 
     Metrics getTopicLoadMetrics() {
-        return getDimensionMetrics("topic_load_times", "topic_load", topicLoadStats);
+        return getDimensionMetrics("brk_topic_load_times", "topic_load", topicLoadStats);
```

Comment:
@Technoboy- Will it break the existing metrics system? As we discussed before, all the metrics changes should be started with a proposal since it might surprise the pulsar operators, and impact the alert system, and metrics system.

Reply:
Ok, I will create a PIP for this.